tests/test_set_config.py

```diff
@@ -40,11 +42,11 @@ def test_path_if_file_exists(): #pylint: disable=unused-variable
     non-existant path.'''
 
     with tempfile.NamedTemporaryFile(dir='.', mode='w') as tmp_file:
-        assert set_config.path_if_file_exists(tmp_file.name)
+        assert cli_helpers.path_if_file_exists(tmp_file.name)
 
     with pytest.raises(argparse.ArgumentTypeError):
         not_a_filepath = './no_way_this_file_exists.nope'
-        set_config.path_if_file_exists(not_a_filepath)
+        cli_helpers.path_if_file_exists(not_a_filepath)
```

Comment:
This should probably go to a test_cli_helpers.py when we get to refactoring the tests